include/motor-control/core/brushed_motor/brushed_motor_interrupt_handler.hpp

```diff
@@ -140,6 +222,10 @@ class BrushedMotorInterruptHandler {
     StatusClient& status_queue_client;
     motor_hardware::BrushedMotorHardwareIface& hardware;
     brushed_motor_driver::BrushedMotorDriverIface& driver_hardware;
+    lms::LinearMotionSystemConfig<lms::GearBoxConfig>& gear_conf;
     BrushedMove buffered_move = BrushedMove{};
+    std::atomic<bool> holding = false;
```

Comment:
also, does it make more sense for this to be an enum of something like
```
enum class State {
   FORCE_CONTROLLING,
   POSITION_CONTROLLING,
   IDLE 
};
```
rather than a `holding` bool and an implicit "don't disable the motor when you finish a force control move"?